views/pages/Search.ejs

```diff
@@ -57,7 +57,7 @@
         <input type="submit" value="Search">
     </form>
 
-    <form action="Search" method="GET">
+    <form action="Searchallrest" method="POST">
```

Comment:
this change isn't needed, just do the following command: git checkout origin/master views/pages/Search.ejs
then commit and push your changes